### PR TITLE
🌐 [i18n-KO] Translated object_detection.mdx to Korean

### PR DESCRIPTION
**Pull request by kihoon71**
# What does this PR do?
Translated the object_detection.mdx file of the documentation to Korean.
Thank you in advance for your review.

Part of https://github.com/huggingface/transformers/issues/20179

## Before reviewing

- [x] Check for missing / redundant translations (번역 누락/중복 검사)
- [x] Grammar Check (맞춤법 검사)
- [x] Review or Add new terms to glossary (용어 확인 및 추가)
- [x] Check Inline TOC (e.g. [[lowercased-header]])
- [x] Check live-preview for gotchas (live-preview로 정상작동 확인)

## Who can review? (Initial)

@0525hhgus, @KIHOON71, @sim-so, @gabrielwithappy, @HanNayeoniee, @wonhyeongseo, @jungnerd

## Before submitting
- [x] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [x] Did you read the [contributor guideline](https://github.com/huggingface/transformers/blob/main/CONTRIBUTING.md#start-contributing-pull-requests),
      Pull Request section?
- [ ] Was this discussed/approved via a Github issue or the [forum](https://discuss.huggingface.co/)? Please add a link
      to it if that's the case.
- [x] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/transformers/tree/main/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/transformers/tree/main/docs#writing-source-documentation).
- [ ] Did you write any new necessary tests?

## Who can review?
May you please review this PR?
@sgugger, @ArthurZucker, @eunseojo

### CODE REVIEW

File: docs/source/ko/tasks/object_detection.mdx [hunk 1]
```diff
@@ -0,0 +1,585 @@
...
+...     targets = [
+...         {"image_id": id_, "annotations": formatted_anns(id_, cat_, ar_, box_)}
+...         for id_, cat_, ar_, box_ in zip(image_ids, categories, area, bboxes)
+...     ]
+
+...     return image_processor(images=images, annotations=targets, return_tensors="pt")
+```
+
+이전 단계에서 만든 전처리 함수를 🤗 Datasets의 [`~datasets.Dataset.with_transform`] 메서드를 사용하여 데이터 세트 전체에 적용합니다.
+이 메서드는 데이터 세트의 요소를 가져올 때마다 전처리 함수를 적용합니다.
```
Comment (0525hhgus) [inline docs/source/ko/tasks/object_detection.mdx:248]:
```suggestion
이 메소드는 데이터 세트의 요소를 가져올 때마다 전처리 함수를 적용합니다.
```
Glossary에 맞춰서 수정했습니다!

File: docs/source/ko/tasks/object_detection.mdx [hunk 2]
```diff
@@ -0,0 +1,585 @@
...
+```
+
+## DETR 모델 학습시키기 [[training-the-DETR-model]]
+
+이전 섹션에서 대부분의 작업을 수행하여 이제 모델을 학습할 준비가 되었습니다!
+이 데이터 세트의 이미지는 리사이즈 후에도 여전히 용량이 크며, 따라서 이 모델을 미세 조정 하려면 적어도 하나의 GPU가 필요합니다.
+
+학습은 다음의 단계를 수행합니다:
+
+1. [`AutoModelForObjectDetection`]을 사용하여 전처리와 동일한 체크포인트를 사용하여 모델을 로드합니다.
```
Comment (0525hhgus) [inline docs/source/ko/tasks/object_detection.mdx:312]:
```suggestion
1. [`AutoModelForObjectDetection`]을 사용하여 전처리와 동일한 체크포인트를 사용하여 모델을 가져옵니다.
```

**Review by 0525hhgus — CHANGES_REQUESTED** [review]
좋은 번역 감사합니다 👍 아래 짧은 리뷰 남겼습니다!

File: docs/source/ko/tasks/object_detection.mdx [hunk 3]
```diff
@@ -0,0 +1,585 @@
...
+- `width`: 이미지의 너비
+- `height`: 이미지의 높이
+- `objects`: 이미지 안의 객체들의 바운딩 박스 메타데이터를 포함하는 딕셔너리:
+  - `id`: 어노테이션 id
+  - `area`: 바운딩 박스의 면적
+  - `bbox`: 객체의 바운딩 박스 ([COCO 포맷](https://albumentations.ai/docs/getting_started/bounding_boxes_augmentation/#coco)으로)
+  - `category`: 객체의 카테고리, 가능한 값으로는 `Coverall (0)`, `Face_Shield (1)`, `Gloves (2)`, `Goggles (3)` 및 `Mask (4)` 가 포함됩니다.
+
+`bbox` 필드가 DETR 모델이 요구하는 COCO 형식을 따른다는 것을 알 수 있습니다. 
+그러나 `objects` 내부의 필드의 그룹화는 DETR이 요구하는 어노테이션 형식과 다릅니다. 따라서 이 데이터를 학습에 사용하기 전에 전처리를 적용해야 합니다.
```
Comment (sim-so) [inline docs/source/ko/tasks/object_detection.mdx:110]:
```suggestion
그러나 `objects` 내부의 필드 그룹은 DETR이 요구하는 어노테이션 형식과 다릅니다. 따라서 이 데이터를 학습에 사용하기 전에 전처리를 적용해야 합니다.
```
`의`가 두 번인 것이 어색하여 수정합니다! 그리고 여기서 `grouping`은 한국어로 `그룹`이 되어도 괜찮을 것 같습니다.

File: docs/source/ko/tasks/object_detection.mdx [hunk 4]
```diff
@@ -0,0 +1,585 @@
...
+사전 훈련된 모델과 동일한 체크포인트에서 이미지 프로세서를 인스턴스화합니다.
+
+```py
+>>> from transformers import AutoImageProcessor
+
+>>> checkpoint = "facebook/detr-resnet-50"
+>>> image_processor = AutoImageProcessor.from_pretrained(checkpoint)
+```
+
+`image_processor`에 이미지를 전달하기 전에, 데이터 세트에 두 개의 전처리를 적용해야 합니다:
```
Comment (sim-so) [inline docs/source/ko/tasks/object_detection.mdx:179]:
```suggestion
`image_processor`에 이미지를 전달하기 전에, 데이터 세트에 두 가지 전처리를 적용해야 합니다:
```

File: docs/source/ko/tasks/object_detection.mdx [hunk 5]
```diff
@@ -0,0 +1,585 @@
...
+
+...     targets = [
+...         {"image_id": id_, "annotations": formatted_anns(id_, cat_, ar_, box_)}
+...         for id_, cat_, ar_, box_ in zip(image_ids, categories, area, bboxes)
+...     ]
+
+...     return image_processor(images=images, annotations=targets, return_tensors="pt")
+```
+
+이전 단계에서 만든 전처리 함수를 🤗 Datasets의 [`~datasets.Dataset.with_transform`] 메서드를 사용하여 데이터 세트 전체에 적용합니다.
```
Comment (sim-so) [inline docs/source/ko/tasks/object_detection.mdx:247]:
```suggestion
이전 단계에서 만든 전처리 함수를 🤗 Datasets의 [`~datasets.Dataset.with_transform`] 메소드를 사용하여 데이터 세트 전체에 적용합니다.
```

File: docs/source/ko/tasks/object_detection.mdx [hunk 6]
```diff
@@ -0,0 +1,585 @@
...
+## DETR 모델 학습시키기 [[training-the-DETR-model]]
+
+이전 섹션에서 대부분의 작업을 수행하여 이제 모델을 학습할 준비가 되었습니다!
+이 데이터 세트의 이미지는 리사이즈 후에도 여전히 용량이 크며, 따라서 이 모델을 미세 조정 하려면 적어도 하나의 GPU가 필요합니다.
+
+학습은 다음의 단계를 수행합니다:
+
+1. [`AutoModelForObjectDetection`]을 사용하여 전처리와 동일한 체크포인트를 사용하여 모델을 로드합니다.
+2. [`TrainingArguments`]에서 학습 하이퍼파라미터를 정의합니다.
+3. 모델, 데이터 세트, 이미지 프로세서 및 데이터 콜레이터와 함께 [`Trainer`]에 학습 인자를 전달합니다.
```
Comment (sim-so) [inline docs/source/ko/tasks/object_detection.mdx:314]:
```suggestion
3. 모델, 데이터 세트, 이미지 프로세서 및 데이터 콜레이터와 함께 [`Trainer`]에 훈련 인수를 전달합니다.
```
glossary에서 `training`은 `훈련`, `arguments`는 `인수`로 정해두었습니다!
문서 내 다른 `training` 등장 부분에도 참고 부탁 드립니다.

File: docs/source/ko/tasks/object_detection.mdx [hunk 7]
```diff
@@ -0,0 +1,585 @@
...
+
+객체 탐지 모델은 일반적으로 일련의 <a href="https://cocodataset.org/#detection-eval">COCO-스타일 지표</a>로 평가됩니다. 
+기존에 구현된 평가지표 중 하나를 사용할 수도 있지만, 여기에서는 허깅페이스 허브에 푸시한 최종 모델을 평가하는 데 `torchvision`에서 제공하는 평가 지표를 사용합니다.
+
+`torchvision` 평가자(evaluator)를 사용하려면 실측값인 COCO 데이터 세트를 준비해야 합니다. 
+COCO 데이터 세트를 빌드하는 API는 데이터를 특정 형식으로 저장해야 하므로, 먼저 이미지와 어노테이션을 디스크에 저장해야 합니다. 
+학습을 위해 데이터를 준비할 때와 마찬가지로, cppe5["test"]에서의 어노테이션은 포맷을 맞춰야 합니다. 그러나 이미지는 그대로 유지해야 합니다.
+
+평가 단계는 약간의 작업이 필요하지만, 크게 세 가지 주요 단계로 나눌 수 있습니다. 
+먼저, cppe5["test"] 세트를 준비합니다: 어노테이션을 포맷에 맞게 만들고 데이터를 디스크에 저장합니다.
```
Comment (sim-so) [inline docs/source/ko/tasks/object_detection.mdx:388]:
```suggestion
먼저, `cppe5["test"]` 세트를 준비합니다: 어노테이션을 포맷에 맞게 만들고 데이터를 디스크에 저장합니다.
```

**Review by sim-so — CHANGES_REQUESTED** [review]
고생 많으셨습니다!
조금 늦었지만 리뷰한 내용을 전달 드립니다 😅

**Review by sim-so — CHANGES_REQUESTED** [review]
고생 많으셨습니다!
조금 늦었지만 리뷰한 내용을 전달 드립니다 😅

File: docs/source/ko/tasks/object_detection.mdx [hunk 8]
```diff
@@ -0,0 +1,585 @@
...
+
+객체 탐지는 이미지에서 인스턴스(예: 사람, 건물 또는 자동차)를 감지하는 컴퓨터 비전 작업입니다. 객체 탐지 모델은 이미지를 입력으로 받고 탐지된 바운딩 박스의 좌표와 관련된 레이블을 출력합니다. 
+하나의 이미지에는 여러 객체가 있을 수 있으며 각각은 자체적인 바운딩 박스와 레이블을 가질 수 있습니다(예: 차와 건물이 있는 이미지). 
+또한 각 객체는 이미지의 다른 부분에 존재할 수 있습니다(예: 이미지에 여러 대의 차가 있을 수 있음). 
+이 작업은 보행자, 도로 표지판, 신호등과 같은 것들을 감지하는 자율 주행에 일반적으로 사용됩니다. 
+다른 응용 분야로는 이미지 내의 객체 수를 계산하거나 이미지 검색 등이 있습니다.
+
+이 가이드에서 다음을 배울 것입니다:
+
+ 1. 합성곱 백본(인풋 데이터의 특성을 추출하는 합성곱 네트워크)과 인코더-디코더 트랜스포머 모델을 결합한 [DETR](https://huggingface.co/docs/transformers/model_doc/detr) 모델을 [CPPE-5](https://huggingface.co/datasets/cppe-5) 데이터 세트에 대해 미세조정 하기.
```
Comment (wonhyeongseo) [inline docs/source/ko/tasks/object_detection.mdx:25]: 아래 방법 대신에 이 줄의 끝점을 없애도 될 것 같아요!
```suggestion
 1. 합성곱 백본(인풋 데이터의 특성을 추출하는 합성곱 네트워크)과 인코더-디코더 트랜스포머 모델을 결합한 [DETR](https://huggingface.co/docs/transformers/model_doc/detr) 모델을 [CPPE-5](https://huggingface.co/datasets/cppe-5) 데이터 세트에 대해 미세조정 하기
```

File: docs/source/ko/tasks/object_detection.mdx [hunk 9]
```diff
@@ -0,0 +1,585 @@
...
+ 2. 미세조정 한 모델을 추론에 사용하기
+
+<Tip>
+이 튜토리얼에서 설명하는 작업은 다음 모델 아키텍처에서 지원됩니다:
```
Comment (wonhyeongseo) [inline docs/source/ko/tasks/object_detection.mdx:29]:
```suggestion
이 튜토리얼의 태스크는 다음 모델 아키텍처에서 지원됩니다:
```

File: docs/source/ko/tasks/object_detection.mdx [hunk 10]
```diff
@@ -0,0 +1,585 @@
...
+    <img src="https://i.imgur.com/TdaqPJO.png" alt="CPPE-5 Image Example"/>
+</div>
+
+바운딩 박스와 연결된 레이블을 시각화하려면 데이터 세트의 메타 데이터, 특히 `category` 필드에서 레이블을 가져와야 합니다. 
+또한 레이블 ID를 레이블 클래스에 매핑하는 `id2label`과 반대로 매핑하는 `label2id` 딕셔너리를 만들어야 합니다. 
+모델을 설정할 때 이러한 매핑을 사용할 수 있습니다. 이러한 매핑은 허깅페이스 허브에서 모델을 공유했을 때 다른 사람들이 재사용할 수 있습니다.
+
+데이터를 더 잘 이해하기 위한 최종 단계로, 잠재적인 문제를 찾아보세요. 
+객체 감지를 위한 데이터 세트에서 자주 발생하는 문제 중 하나는 바운딩 박스가 이미지의 가장자리를 넘어가는 것입니다. 
+이러한 바운딩 박스를 "넘어가는 것(run away)"은 훈련 중에 오류를 발생시킬 수 있기에 이 단계에서 처리해야 합니다.
```
Comment (wonhyeongseo) [inline docs/source/ko/tasks/object_detection.mdx:148]: 아주 좋아요 👍

**Review by wonhyeongseo — CHANGES_REQUESTED** [review]
좋은 번역 감사합니다! 💯

**Review by ArthurZucker — APPROVED** [review]
Congrats on getting this long one done! 🔥 
Thanks!

**Review by sgugger — APPROVED** [review]

**Pull request merged**